circuits-circom/regexes/venmo/venmo_p2p_check.json

```diff
@@ -0,0 +1,9 @@
+{
+	"parts": [
+		{
+			"is_public": false,
+			"regex_def": "liable for non-deliver=\r\ny or delayed"
```

Comment:
A smaller regex is fine because here we are more concerned about preventing a merchant email to pass through than ensuring an email is a merchant email.